eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -50,9 +50,9 @@ jobs:
     # Linux musl x64
     - ${{ if eq(parameters.platform, 'linux_musl_x64') }}:
       - ${{ if or(ne(parameters.jobParameters.isExtraPlatforms, true), eq(parameters.jobParameters.includeAllPlatforms, true)) }}:
-        - (Alpine.314.Amd64.Open)Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.14-helix-amd64
+        - (Alpine.315.Amd64.Open)Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.15-helix-amd64-staging
```

Comment:
Do we still want to put staging here? Are the images still not available?
If so, I think we should file an issue to remove the staging tag once they are available.

Reply:
no, they are not. They should be available today or tomorrow. 
However, they happen only once a week. I somewhat expect more changes in msquic to support OpenSSL 3 so consuming production tags will impact our ability to consume them quickly. 

I'm fine with eater direction, please let me know how you want to proceed @ManickaP.

Reply:
This should get rolled out soon enough. Apart from that, it also affects everyone else and not just us. ==> I'd leave the default here. 
In case we really need to test something immediately, we can always temporarily switch the image in that particular PR, just for testing.